mypyc/ops.py

```diff
@@ -1550,6 +1548,17 @@ def has_method(self, name: str) -> bool:
             return False
         return True
 
+    def is_method_final(self, name: str) -> bool:
+        if self.has_method(name):
+            method_decl = self.method_decl(name)
+            for subc in self.subclasses():
+                if subc.method_decl(name) != method_decl:
+                    return False
+            return True
+        else:
+            return not any(name in subc.method_decls
```

Comment:
This is the fix (checking for `method_decls` as it's too early to look for `method`s)

Reply:
Actually, I think this else branch is wrong,  I  need the approach from the other branch. Will fix